.gitignore

```diff
@@ -7,4 +7,5 @@
 *.iml
 *.ipr
 
-laundry-management-service/api/src/main/java/com/microservices/laundrymanagementapi/messages/
\ No newline at end of file
+#pattern for excluding generated proto files from git
+**/*api/messages/
```

Comment:
`**/api/messages/` - in case you've renamed your package (and you did)

Reply:
fixed